framework/src/modules/dpos/transaction_assets/pom_transaction_asset.ts

```diff
@@ -230,10 +230,19 @@ export class PomTransactionAsset extends BaseAsset<PomTransactionAssetContext> {
 		const delegateAccountBalance = await reducerHandler.invoke<bigint>('token:getBalance', {
 			address: delegateAccount.address,
 		});
+		const minRemainingBalance = await reducerHandler.invoke<bigint>(
+			'token:getMinRemainingBalance',
+			{},
```

Comment:
Why its necessary to pass empty object? As we got a use case we should make second argument optional.